enigma-core/app/src/networking/ipc_listener.rs

```diff
@@ -353,9 +351,16 @@ pub(self) mod handling {
                 // Save the ExeCode into the DB.
                 let key = DeltaKey::new(contract_address, Stype::ByteCode);
                 db.create(&key, &v.output)?;
-                Ok(v.into_deploy_response(&bytecode))
+                let wasm_result = v.into_deploy_response(&bytecode);
+                info!("deploy_contract() => Ok({})", wasm_result.display_without_bytecode());
+                debug!("deploy_contract() => {}", wasm_result.display_bytecode());
```

Comment:
```suggestion
                debug!("bytecode => {}", wasm_result.display_bytecode());
```

Reply:
I don't like that we have two separate outputs that contain the deploy_contract signature, it might be preferable to structure it in this way. just a thought, feel free to decide whatever..